### PR TITLE
chore: suppress blib2to3.pgen2.driver output to log

### LINKED ISSUE #149: Suppress blib2to3.pgen2.driver output...
After adding `format_code` support in #144, there's a lot of debugging output like `blib2to3.pgen2.driver > Stop.` that is littered over the logs, cluttering that doesn't help with debugging through the logs. Will investigate to suppress this 😢

### PR DESCRIPTION
**Pull request by dandhlee**
Verified that the outputs are indeed suppressed in local test.

Fixes #149.

- [x] Tests pass

### CODE REVIEW

**Review by nicain — APPROVED** [review]
LGTM

**Comment by dandhlee** [comment]
@nicain sob why are you not yoshi-python 😢

File: docfx_yaml/extension.py [hunk 1]
```diff
@@ -24,6 +24,11 @@
 import copy
 import shutil
 import black
+
+import logging
```
Comment (tbpg) [inline docfx_yaml/extension.py:28]: Put this with the other imports. Move the `getlogger` call to below the imports.

Comment (dandhlee) [reply docfx_yaml/extension.py:27]: Done.

**Comment by dandhlee** [comment]
Please take a look again!

**Review by busunkim96 — APPROVED** [review]

**Review by tbpg — APPROVED** [review]

**Pull request merged**